src/Diagnostics.DataProviders/DataProviders/MockSupportObserverDataProvider.cs

```diff
@@ -20,21 +20,16 @@ public MockSupportObserverDataProvider(OperationDataCache cache, SupportObserver
         {
         }
 
-        public override Task<Dictionary<string, List<RuntimeSitenameTimeRange>>> GetRuntimeSiteSlotMap(string siteName)
```

Comment:
Lets find out how many detectors are using this signature and then convert them to using the `GetRuntimeSiteSlotMap(string stampName, string siteName)` variant

Reply:
I don't see any using this method - this is a precaution so that users don't run into it, but I could also remove it from the API

Reply:
Yea I think we can just remove this then